minichain/p2p.py

```
@@ -15,16 +16,33 @@
+# Misbehavior thresholds — all four are overridable per P2PNetwork instance.
+MALFORMED_THRESHOLD = 15     # N: accumulated malformed messages before ban
+FAILED_THRESHOLD = 15        # M: accumulated failed messages before ban
+INVALID_THRESHOLD = 1        # L: accumulated invalid messages before ban (1 = immediate)
+DECAY_INTERVAL_MINUTES = 10  # T: counter half-life period in minutes
+
+
 class P2PNetwork:
     """Lightweight peer-to-peer networking using libp2p."""
 
-    def __init__(self, handler_callback=None):
+    def __init__(
```

Ruff (0.15.18)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)